packages/mediapipe-task-text/lib/src/mediapipe_text_bindings.dart

```diff
@@ -0,0 +1,292 @@
+
+  ffi.Pointer<TextClassifierResult> allocate_text_classifier_result() {
+    return _allocate_text_classifier_result();
+  }
+
+  late final _allocate_text_classifier_resultPtr =
+      _lookup<ffi.NativeFunction<ffi.Pointer<TextClassifierResult> Function()>>(
+          'allocate_text_classifier_result');
+  late final _allocate_text_classifier_result =
+      _allocate_text_classifier_resultPtr
+          .asFunction<ffi.Pointer<TextClassifierResult> Function()>();
+
+  ffi.Pointer<ffi.Void> text_classifier_create(
+    ffi.Pointer<TextClassifierOptions> options,
+  ) {
+    return _text_classifier_create(
+      options,
+    );
+  }
+
+  late final _text_classifier_createPtr = _lookup<
+      ffi.NativeFunction<
+          ffi.Pointer<ffi.Void> Function(
+              ffi.Pointer<TextClassifierOptions>)>>('text_classifier_create');
+  late final _text_classifier_create = _text_classifier_createPtr.asFunction<
+      ffi.Pointer<ffi.Void> Function(ffi.Pointer<TextClassifierOptions>)>();
+
+  void text_classifier_classify(
+    ffi.Pointer<ffi.Void> classifier,
+    ffi.Pointer<TextClassifierResult> results,
+    ffi.Pointer<ffi.Char> utf8_text,
+  ) {
+    return _text_classifier_classify(
+      classifier,
+      results,
+      utf8_text,
+    );
+  }
+
+  late final _text_classifier_classifyPtr = _lookup<
+      ffi.NativeFunction<
+          ffi.Void Function(
+              ffi.Pointer<ffi.Void>,
+              ffi.Pointer<TextClassifierResult>,
+              ffi.Pointer<ffi.Char>)>>('text_classifier_classify');
+  late final _text_classifier_classify =
+      _text_classifier_classifyPtr.asFunction<
+          void Function(ffi.Pointer<ffi.Void>,
+              ffi.Pointer<TextClassifierResult>, ffi.Pointer<ffi.Char>)>();
+
+  void text_classifier_close(
+    ffi.Pointer<ffi.Void> classifier,
+  ) {
+    return _text_classifier_close(
+      classifier,
+    );
+  }
+
+  late final _text_classifier_closePtr =
+      _lookup<ffi.NativeFunction<ffi.Void Function(ffi.Pointer<ffi.Void>)>>(
+          'text_classifier_close');
+  late final _text_classifier_close = _text_classifier_closePtr
+      .asFunction<void Function(ffi.Pointer<ffi.Void>)>();
+
+  void text_classifier_result_close(
+    ffi.Pointer<TextClassifierResult> result,
+  ) {
+    return _text_classifier_result_close(
+      result,
+    );
+  }
+
+  late final _text_classifier_result_closePtr = _lookup<
+          ffi.NativeFunction<
+              ffi.Void Function(ffi.Pointer<TextClassifierResult>)>>(
+      'text_classifier_result_close');
+  late final _text_classifier_result_close = _text_classifier_result_closePtr
+      .asFunction<void Function(ffi.Pointer<TextClassifierResult>)>();
+}
+
+final class TextClassifierOptions extends ffi.Struct {
+  external imp1.BaseOptions base_options;
+
+  external imp1.ClassifierOptions classifier_options;
+}
+
+typedef TextClassifierResult = imp1.ClassificationResult;
+
+const int true1 = 1;
```

Comment:
What are these super fun values? Maybe I am missing context

Reply:
I think the context is that C is bad at booleans 😆